database/factories/FeedbackFactory.php

```diff
@@ -2,12 +2,12 @@
 
 namespace Database\Factories;
 
-use App\Enums\Feedback;
-use App\Models\User;
+use App\Domain\Feedbacks\Enums\Feedback;
```

Comment:
There is no plural of "feedback" as it is "feedback". Maybe we can stick to proper English 😅

Reply:
Good catch!